content_sync/models.py

```diff
@@ -1,4 +1,32 @@
-"""Content sync models"""
-# from django.db import models
+""" Content sync models """
+from django.db import models
 
-# Create your models here.
+
+class ContentSyncState(models.Model):
```

Comment:
Do you think it's worth adding this to Django admin?

Reply:
I don't think there should be a need to work with this model in django-admin, since all of its fields shouldn't be modified outside of code.